### PR TITLE
Added Issue Template and Pull Request template

### PR DESCRIPTION
**Pull request by deytulsi18**
Fixes: #20
Added Issue Template and Pull Request template, PR with reference to

### CODE REVIEW

**Comment by deytulsi18** [comment]
Please review and merge the PR.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: This is unnecessary, you can remove this (line 1 to 18)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: Bug report
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
** Steps to reproduce the bug**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Steps to reproduce the behavior:
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: Remove this line

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+1. 
+2. 
+3. 
+4. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: write w.r.t in full words (with respect to)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: Same instructions for this template

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (kaustubhgupta) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]: You can remove this header and associated text with this (next line)

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- issue goes here
```
Comment (kaustubhgupta) [inline .github/pull_request_template.md:2]:
```suggestion
- Fixes: #
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+# Prosposed Changes
+- change 1
+- change 2
+
+# Additional Info
+- any additional information or context
+
+# Checklist
```
Comment (kaustubhgupta) [inline .github/pull_request_template.md:11]: For now, remove this checklist

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Tests
+- [ ] Translations
+- [ ] Documentations
+
+# Screenshots
+
+Original             |  Updated
```
Comment (kaustubhgupta) [inline .github/pull_request_template.md:18]: Remove this table, only screenshots header is fine

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by deytulsi18** [comment]
OK, making the necessary changes

**Comment by deytulsi18** [comment]
Kindly review the newly made changes.

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+# Related Issue
+- Fixes: #
+
+# Prosposed Changes
+- change 1
+- change 2
+
+# Screenshots
+
+Original             |  Updated
```
Comment (kaustubhgupta) [inline .github/pull_request_template.md:10]: Remove this line too

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@deytulsi18 Have a look at the new requested changes if you haven't already

**Comment by kaustubhgupta** [comment]
Due to inactivity, this PR is closed. Also, I no longer need any template for this repo!

**Pull request closed without merging**